dotcom-rendering/cypress/lib/mockDate.js

```diff
@@ -0,0 +1,19 @@
+export class MorningDate extends Date {
```

Comment:
Didn't use `cy.clock` as I couldn't get that to work. With this being just a test for an ab test I think it's ok to do it this more rudimentary way